.github/workflows/validator.yml

```diff
@@ -0,0 +1,72 @@
+
+name: Dataset Validator
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  Validate:
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+
+    #- name: Update software
+    #  run: |
+    # do we want to do this? it's helpful but also slow.
+    #    sudo apt-get update &&
+    #    sudo DEBIAN_FRONTEND=noninteractive apt-get -y dist-upgrade &&
+    #    sudo DEBIAN_FRONTEND=noninteractive apt-get autoremove
```

Comment:
It's common practice to start building environments by running system updates. I've seen it in Dockerfiles and many CI scripts. It helps ensure your local dev environment and the CI environment are in sync, and that any bugs are as fixed as they possibly can be.

But it also takes 2-3 minutes of CI time and I am impatient. The git-annex and other CLI tools available on the Github machines seem to be new enough to pass our tests so I left it out.

Reply:
```suggestion
    #- name: Update software
    #  run: |
    #    # do we want to do this? it's helpful to avoid testing against surprise out-of-date software, but also so slow.
    #    sudo apt-get update &&
    #    sudo DEBIAN_FRONTEND=noninteractive apt-get -y dist-upgrade &&
    #    sudo DEBIAN_FRONTEND=noninteractive apt-get autoremove
```